words_dir/tweet_historical_events.py

```diff
@@ -11,10 +11,12 @@
 # NOTE: must fit in a tweet, so not longer than this line -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 
 events = {
+    
     #1600s
     '1629-01-01': 'In 1629, the General Court of Colonial Virginia met to decide if a certain settler was male or female. https://www.history.com/news/intersex-people-casimir-pulaski-thomasine-hall',
 
     # 1800s
+    '1868-05-06': 'The word "homosexuality" was not coined until 1868 by German-Hungarian Karoly Maria Kertbeny (who advocated decriminalization). https://en.wikipedia.org/wiki/LGBT_history'
```

Comment:
Please add a comma to the end of the line. If you don't, the project will throw an error or do some goofy stuff like smashing together this line with the next line